packages/connect/src/core/onCallFirmwareUpdate.ts

```diff
@@ -165,11 +171,18 @@ const getInstallationParams = (device: Device, params: Params) => {
             device.atLeast('2.6.3') && isUpdatingToNewerVersion && isUpdatingToEqualFirmwareType;
         const manual = !device.atLeast(['1.10.0', '2.6.0']) && !upgrade;
 
+        const getUpdateFlowType = (): FirmwareUpdateFlowType => {
+            if (manual) return 'manual';
+
+            return upgrade ? 'reboot_and_upgrade' : 'reboot_and_wait';
+        };
+
         return {
             /** RebootToBootloader is not supported */
             manual,
             /** RebootToBootloader (REBOOT_AND_UPGRADE) is supported  */
```

Comment:
Observation: I know this was not changed in this PR but since now we have `reboot_and_upgrade` as string in type `FirmwareUpdateFlowType` wouldn't it make sense to change it also to be non capital?

Reply:
For example when I see `undefined` I might not be sure if it is legit `undefined` or just the attribute in the object did not get assigned for some reason.